test/integration/backward_compatible/HazelcastClientTest.js

```diff
@@ -116,44 +116,31 @@ configParams.forEach(function (cfg) {
             expect(info.labels).to.deep.equal(new Set());
         });
 
-        it('getDistributedObjects returns all dist objects', function (done) {
+        it('getDistributedObjects returns all dist objects', function () {
             managed.getObject(client.getMap.bind(client, 'map'));
             managed.getObject(client.getSet.bind(client, 'set'));
-            setTimeout(function () {
-                client.getDistributedObjects().then(function (distObjects) {
-                    try {
-                        const names = distObjects.map((o) => {
-                            return o.getName();
-                        });
-                        expect(names).to.have.members(['map', 'set']);
-                        done();
-                    } catch (e) {
-                        done(e);
-                    }
+            return TestUtil.assertTrueEventually(async function () {
+                const distObjects = await client.getDistributedObjects();
+                const names = distObjects.map((o) => {
+                    return o.getName();
                 });
-            }, 300);
+                expect(names).to.have.members(['map', 'set']);
+            }, 100, 5000);
         });
 
-        it('getDistributedObjects does not return removed object', function (done) {
+        it('getDistributedObjects does not return removed object', function () {
             managed.getObject(client.getMap.bind(client, 'map1'));
             managed.getObject(client.getMap.bind(client, 'map2'));
             managed.getObject(client.getMap.bind(client, 'map3'));
 
-            setTimeout(function () {
-                managed.destroy('map1').then(function () {
-                    client.getDistributedObjects().then(function (distObjects) {
-                        try {
-                            const names = distObjects.map(function (o) {
-                                return o.getName();
-                            });
-                            expect(names).to.have.members(['map2', 'map3']);
-                            done();
-                        } catch (e) {
-                            done(e);
-                        }
-                    });
+            return TestUtil.assertTrueEventually( async function () {
```

Comment:
Same comment as above. Also, there is an unnecessary space in `TestUtil.assertTrueEventually( async`.

Reply:
>Also, there is an unnecessary space in TestUtil.assertTrueEventually( async.
To avoid this, I plan to add `"space-in-parens": "error"` to root eslint config.(no error with current code base)


ref: https://eslint.org/docs/rules/space-in-parens

Reply:
Sounds good to me